lang/en_US.json

```diff
@@ -660,7 +660,7 @@
     "New username must be different from current username.": "New username must be different from current username.",
     "Your username change request is being reviewed.": "Your username change request is being reviewed.",
     "Your username cannot be changed right now.": "Your username cannot be changed right now.",
-    "You can request another change after your previous request's 30-day cooldown period has ended.": "You can request another change after your previous request's 30-day cooldown period has ended.",
+    "You can request another change after your previous request's cooldown period has ended.": "You can request another change after your previous request's cooldown period has ended.",
```

Comment:
I wonder if it would be worth including the cooldown times here. After a request has been submitted, the user has no idea how much longer they have to wait.